docs/how-to/installation/aws.txt

```diff
@@ -359,6 +359,8 @@ Master
 -  Allow EC2 to assume role
 -  Allow EC2 to dynamically create and terminate instances with agent
    role
+-  Allow EC2 to create and terminate spot instance requests (only
```

Comment:
blocking: should be "Allow EC2 to describe, create and terminate spot instance requests (only"

Reply:
Done